src/jit/importer.cpp

```diff
@@ -7102,33 +7120,32 @@ GenTreePtr          Compiler::impFixupStructReturnType(GenTreePtr op, CORINFO_CL
 #if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
 
 #ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
-    // No VarArgs for CoreCLR.
-    assert(!info.compIsVarArgs); 
+    // No VarArgs for CoreCLR on x64 Unix
```

Comment:
I thought it was the case that coreclr, in general, did not support varargs. If that is the case, this comment should be left as-is. If not, is this something that has a work item associated with it?